lms/envs/common.py

```diff
@@ -1221,6 +1224,12 @@
     sorted(rooted_glob(PROJECT_ROOT / 'static', 'coffee/src/modules/**/*.js'))
 )
 
+proctoring_js = (
+    ['proctoring/js/models/*.js'] +
```

Comment:
I don't see these files in this PR (and they also don't seem to be in master). Where are they?